contrib/inmemory_export/build.gradle

```diff
@@ -0,0 +1,7 @@
+description = 'OpenTelemetry InMemory Export'
```

Comment:
I prefer to have this in contrib_sdk directory. Contrib is a directory that just depends on the API and contrib_sdk depends on the API + SDK :)